### ISSUE TITLE
Incorrect time tracking phpstorm

### ISSUE THREAD

**Issue by sanasol** [wakatime/jetbrains-wakatime#183]
> PhpStorm 2021.2
> Build #PS-212.4746.100, built on July 28, 2021
> Subscription is active until August 27, 2021.
> Runtime version: 11.0.11+9-b1504.13 x86_64
> VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
> macOS 11.5.2
> GC: G1 Young Generation, G1 Old Generation
> Memory: 2048M
> Cores: 12
> Registry: run.processes.with.pty=TRUE, ide.balloon.shadow.size=0
> Non-Bundled Plugins: com.wakatime.intellij.plugin (13.0.2), zielu.gittoolbox (212.7.6), com.deadlock.scsyntax (212.4746.57), de.espend.idea.laravel (0.15.4), ru.adelf.idea.dotenv (2021.3.0.212)

Noticed it about ~2 weeks ago(since 13 august).
IDLE time tracked like work with random opened file.

Time tracked while phpstorm not focused even.
Also probably problem is getting worse when opened multiple projects at once.

Looks like it in worst case:
![image](https://user-images.githubusercontent.com/1709666/130775482-f6e02320-9502-42ed-a24d-9855fd295239.png)

Here is log with wakatime debug enabled sometimes.
[idea.log](https://github.com/wakatime/jetbrains-wakatime/files/7046243/idea.log)

Checked idea.log and I see that possible problem is GitToolBox plugin, WakaTime triggered every time autofetch or blame executed. But it wasn't problem before. So probably should be some fix.

**Comment by alanhamlett** [comment]
Yes, `Saving Project` sounds like it would trigger the on-save event that WakaTime listens for:

```
42661 2021-08-25 13:18:42,176 [1902703]   INFO - rationStore.ComponentStoreImpl - Saving Project(name=myproject, containerState=COMPONENT_CREATED, componentStore=/Users/user/code/lpt-v1)GitToolBoxBlamePersistence took 14 ms```

I'll install that plugin and see if we can ignore these events.
